src/BuildScriptGenerator/DefaultRunScriptGenerator.cs

```diff
@@ -0,0 +1,64 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.Extensions.Logging;
+using Microsoft.Oryx.BuildScriptGenerator.Exceptions;
+using Microsoft.Oryx.Common;
+
+namespace Microsoft.Oryx.BuildScriptGenerator
+{
+    internal class DefaultRunScriptGenerator : IRunScriptGenerator
```

Comment:
This should be showing as a rename: https://github.com/microsoft/Oryx/pull/194/commits/f49586d3346b672127cde8fbe39ba349365e61ae